logger/pionlogger/logadapter.go

```diff
@@ -28,11 +28,11 @@ type logAdapter struct {
 }
 
 func (l *logAdapter) Trace(msg string) {
-	// ignore trace
+	l.logger.Debugw(msg)
 }
 
 func (l *logAdapter) Tracef(format string, args ...interface{}) {
-	// ignore trace
```

Comment:
Should we do the l.shouldIgnore check with these?

Reply:
Intentional skip as we want verbose details.